fleet/lib/multus-demo/fleet.yaml

```diff
@@ -47,3 +48,4 @@ targetCustomizations:
         networks:
           - dds
           - lhn
+          - ra-lhn
```

Comment:
Isn't this going to cause an IP address conflict on yagan if a user pod consumes the net-attach-def?

Reply:
yes, it is going to! 😱  there is only 1 IP address, should i leave the example just in ruka for testing purposes?

Reply:
created a demo net-attch-def and used that with the next IP in the segment